BRITECH-72 Fetching correspondence from dynamodb

dwybourn opened the pull request.

lclarksl commented on README.md:
```diff
@@ -16,8 +18,35 @@ The date is in iso-8601 format.
+    "bob.builder@cartoonconstructionslimited.tv" : "2004-01-30 05:00:01",
+    "chris.smith@leaddeveloper.com" : "2016-02-17 15:51:15",
+    "david.wybourn@superawesomegoodcode.co.uk" : "2016-03-07 12:45:04"
+}
```
If we are storing datetimes in ISO 8601, it should be in this format 2016-03-07T12:45:04Z for UTC time

jebbinSCL commented:
I wonder if until we have larger services and controllers we should maybe have a single Services.fs and Controller.fs? At the moment, the actual code is only one or two lines for the controllers, and the services are quite small as well. @csscottlogic ? 

Also, want to install the F# power tools for visual studio and make sure you have all of the syntax colouring options enabled? You have a few unused references in some of the files.

jebbinSCL commented:
Also, want to install the F# power tools for visual studio and make sure you have all of the syntax colouring options enabled? You have a few unused references in some of the files.

lclarksl commented on Speaker-comms/CorrespondenceController.fs:
```diff
@@ -0,0 +1,10 @@
+    inherit ApiController()
+
+    member __.Get() = 
+        getCorrespondence()
```
It might be worth considering wrapping this in Request.CreateResponse(). Although, maybe we should wait until we want to return other HTTP responses.

dwybourn replied:
What does Request.CreateResonse() provide us? Reading online it appears that a lot of people do this, but I can't see why.

lclarksl replied:
I don't think it makes a difference in this case. It means you can do things like Request.CreateResponse(HttpStatusCode.NotFound) when we want to return a 404 and we'd want all our responses to be returned in the same way for consistency. But we don't yet, so we could leave it as is for now,

lclarksl commented:
:shipit: apart from dates

dwybourn commented:
Dates on the readme have been changed back to iso-8601. Added a Request.CreateResponse wrapper on the controller. Removed unused references and variables.

lclarksl commented:
CorrespondenceController should have Request.CreateReponse too.

Then :shipit:

csscottlogic commented:
I'm happy with either a file per controller or a Controller.fs file.
